### ISSUE TITLE
Re-calc will not reduce balance if transaction is moved or deleted.

### ISSUE THREAD

**Issue by akusav** [TampereHacklab/mulysa#410]
If transaction is moved or deleted re-calc should reduce service balance.

**Comment by vranki** [comment]
I don't think this is possible with current model. Recalc doesn't know history and history can be altered by hand by admins. That's why it can only advance paid time according to new, unprocessed payments. Not saying this is impossible, but it would need rethinking the whole model.

**Comment by akusav** [comment]
Ok so let's just keep in mind. That function is only for adding balance and
reducing have to be maid manually.
